$OpenBSD: patch-lib_awful_client_lua_in,v 1.5 2012/08/21 10:09:57 dcoppa Exp $

Fix client screen in awful.client.setslave
(upstream git commit 99230fc632bec31c776c483f61c8620af028bbb6)

--- lib/awful/client.lua.in.orig	Tue Aug 21 12:00:56 2012
+++ lib/awful/client.lua.in	Tue Aug 21 12:01:42 2012
@@ -405,7 +405,7 @@ end
 --- Set the client as slave: put it at the end of other windows.
 -- @param c The window to set as slave.
 function setslave(c)
-    local cls = capi.client.get(screen)
+    local cls = capi.client.get(c.screen)
     for k, v in pairs(cls) do
         c:swap(v)
     end
